### PR TITLE
Change amd64 Dockerfile to ubi

### PR DESCRIPTION
**Pull request by lmm**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
Calico component images are now using registry.access.redhat.com/ubi8/ubi-minimal:latest for the base image
```

### CODE REVIEW

File: Dockerfile.amd64 [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ARG GIT_VERSION=unknown
+
+LABEL name="Calico pod2daemon" \
+      vendor="Project Calico" \
+      version=$GIT_VERSION \
+      release="1" \
+      summary="Secure connections from K8s Pods to local daemons using FlexVolume + Unix Domain Socket" \
+      description="Secure connections from K8s Pods to local daemons using FlexVolume + Unix Domain Socket" \
```
Comment (lmm) [inline Dockerfile.amd64]: Thoughts on the description / name?

File: Dockerfile.amd64 [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ARG GIT_VERSION=unknown
+
+LABEL name="Calico pod2daemon" \
```
Comment (tmjd) [inline Dockerfile.amd64]: I don't know, maybe "Calico FlexVolume driver installer"

File: Dockerfile.amd64 [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+      vendor="Project Calico" \
+      version=$GIT_VERSION \
+      release="1" \
+      summary="Secure connections from Kubernetes pods to local daemons using FlexVolume + Unix Domain Socket" \
```
Comment (tmjd) [inline Dockerfile.amd64]: I'd instead maybe say `FlexVolume installer to setup secure connections from Kubernetes pods to local daemons`.
IDK, maybe

File: Dockerfile.amd64 [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+# limitations under the License.
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ARG GIT_VERSION=unknown
+
+LABEL name="Calico pod2daemon" \
+      vendor="Project Calico" \
+      version=$GIT_VERSION \
+      release="1" \
```
Comment (tmjd) [inline Dockerfile.amd64]: Same as other PR, should we be setting this.

File: Dockerfile.amd64 (hunk 3 above)
Comment (lmm) [reply Dockerfile.amd64]: That is better, thanks! 19cd0ab

File: Dockerfile.amd64 (hunk 4 above)
Comment (lmm) [reply Dockerfile.amd64]: I've been looking at some of the other images in the RH registry and some have just set it to `release=1`  across different tags. I do not think this value affects certification. I've reached out to RH for clarification but I think what we have here should be fine.

Comment (lmm) [reply Dockerfile.amd64]: I reached out to RH and they've confirmed that this label is for partner use.

**Pull request merged**